code/modules/mining/machine_vending.dm

```diff
@@ -35,6 +37,7 @@
 		new /datum/data/mining_equipment("First-Aid Kit",				/obj/item/storage/firstaid/regular,											800),
 		new /datum/data/mining_equipment("Advanced Scanner",			/obj/item/t_scanner/adv_mining_scanner,								800),
 		new /datum/data/mining_equipment("Resonator",					/obj/item/resonator,												800),
+		new /datum/data/mining_equipment("Mini Extinguisher",			/obj/item/extinguisher/mini,										1000),
```

Comment:
1000 points for something that can be easily printed off in cargo?
200 would be more then fine for it

Reply:
If they wanna take the easy way they can. This is just adding it to the vendor because it makes sense, and the vendor already has some inside it (when redeeming one of the voucher options).